apps/gromacs/amis/amzn2-pc-gromacs/amzn2-pc-gromacs.json

```diff
@@ -0,0 +1,144 @@
+          "architecture": "x86_64",
+          "root-device-type": "ebs"
+        },
+        "owners": [
+          "amazon"
+        ],
+        "most_recent": true
+      },
+      "ssh_username": "ec2-user",
+      "ami_name": "{{user `company_name`}}-{{user `name`}}-{{ user `parallel_cluster_version`}}-{{user `application_name`}}-{{user `application_version`}}-{{isotime \"20060102150405\" | clean_resource_name}}",
+      "shutdown_behavior": "terminate",
+      "associate_public_ip_address": "{{user `public_ip`}}",
+      "encrypt_boot": "{{user `encrypt_boot`}}",
+      "tags": {
+          "Name": "{{user `company_name`}}-{{user `name`}}-{{user `application_name`}}",
+          "State": "{{user `state`}}",
+          "Env": "{{user `env`}}",
+          "OS": "Amazon Linux 2",
+          "Source_AMI": "{{ .SourceAMI }}"
+      },
+      "ssh_interface": "{{user `ssh_interface`}}",
+      "communicator": "ssh",
+      "launch_block_device_mappings": [
+        {
+          "device_name": "/dev/xvda",
+          "volume_size": 40,
+          "delete_on_termination": true
+        }
+      ],
+      "temporary_iam_instance_profile_policy_document": {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:DescribeAssociation",
+              "ssm:GetDeployablePatchSnapshotForInstance",
+              "ssm:GetDocument",
+              "ssm:DescribeDocument",
+              "ssm:GetManifest",
+              "ssm:GetParameter",
+              "ssm:GetParameters",
+              "ssm:ListAssociations",
+              "ssm:ListInstanceAssociations",
+              "ssm:PutInventory",
+              "ssm:PutComplianceItems",
+              "ssm:PutConfigurePackageResult",
+              "ssm:UpdateAssociationStatus",
+              "ssm:UpdateInstanceAssociationStatus",
+              "ssm:UpdateInstanceInformation"
+            ],
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssmmessages:CreateControlChannel",
+              "ssmmessages:CreateDataChannel",
+              "ssmmessages:OpenControlChannel",
+              "ssmmessages:OpenDataChannel"
+            ],
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ec2messages:AcknowledgeMessage",
+              "ec2messages:DeleteMessage",
+              "ec2messages:FailMessage",
+              "ec2messages:GetEndpoint",
+              "ec2messages:GetMessages",
+              "ec2messages:SendReply"
+            ],
+            "Resource": "*"
+          }
+        ]
+      },
+      "pause_before_ssm": "30s"
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "shell",
+      "inline": [
+        "sudo yum -y update --security",
+        "sudo yum remove -y dpkg"
+      ]
+    },
+    {
+      "type": "shell",
```

Comment:
Remove the modules install. It is already part of the Intel oneAPI install.

Reply:
done